src/vs/editor/contrib/hover/browser/modesContentHover.ts

```diff
@@ -245,24 +242,7 @@ export class ModesContentHoverWidget extends ContentHoverWidget {
 			msg.contents
 				.filter(contents => !!contents)
 				.forEach(contents => {
-					const renderedContents = renderMarkedString(contents, {
-						actionCallback: (content) => {
-							this._openerService.open(URI.parse(content)).then(void 0, onUnexpectedError);
-						},
-						codeBlockRenderer: (languageAlias, value): string | TPromise<string> => {
-							// In markdown,
-							// it is possible that we stumble upon language aliases (e.g.js instead of javascript)
-							// it is possible no alias is given in which case we fall back to the current editor lang
-							const modeId = languageAlias
-								? this._modeService.getModeIdForLanguageName(languageAlias)
-								: this._editor.getModel().getModeId();
-
-							return this._modeService.getOrCreateMode(modeId).then(_ => {
-								return `<div class="code">${tokenizeToString(value, modeId)}</div>`;
-							});
-						}
-					});
-
+					const renderedContents = renderMarkedString(this._modeService, this._openerService, contents);
```

Comment:
We have a feature that we use the documents language when a code block doesn't specify one (just triple tick pairs). We are loosing that feature here